python/lsst/meas/astrom/astrometry.py

```diff
@@ -106,12 +106,19 @@ class AstrometryConfig(RefMatchConfig):
         default=0.3,
         optional=True,
     )
+    filterMap = pexConfig.DictField(
+        doc="Mapping from physicalFilter label to reference filter name.",
```

Comment:
```suggestion
        doc="Mapping from physical filter label to reference filter name.",
```
(no need to pick a side on camelCase vs. snake_case in prose for humans!)